components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -169,62 +285,91 @@ service NodeManagerService {
 message ConnectResponse{}
 
 message Id {
+	// UUID for the nodemanager.
 	string id = 1;
 }
 message Ids {
+	// List of nodemanager UUIDs.
 	repeated Id ids = 1;
 }
 
 message Query {
+	// Filters to be applied to the query.
 	repeated chef.automate.domain.compliance.api.common.Filter filter_map = 25;
-
+	// Sort the results in ascending or descending order.
 	enum OrderType {
 		ASC = 0;
 		DESC = 1;
 	}
 	OrderType order = 21;
+	// Field to sort on.
 	string sort = 22;
+	// Page number of results to return.
 	int32 page = 23;
+	// Count of results that should be returned for each page.
 	int32 per_page = 24;
 }
 
 message Fields {
+	// List of available fields matching the requested criteria.
 	repeated string fields = 1;
 }
 
 message FieldQuery {
-		Query query = 20;
-		string field = 21;
-		string node_manager_id = 22;
+	// Query details (filters) to be applied to the results.
+	Query query = 20;
+	// Field to search on.
+	string field = 21;
+	// Nodemanager id for which the search is being made.
+	string node_manager_id = 22;
 }
 
 message Nodes {
-		repeated string nodes = 1;
-		int32 total = 20;
+	// List of node names matching the request.
+	repeated string nodes = 1;
+	// Total count of node names matching the request.
+	int32 total = 20;
 }
 
 message NodeQuery {
-		Query query = 20;
-		string node_manager_id = 21;
+	// Query details (filters) to be applied to the results.
+	Query query = 20;
```

Comment:
The same comment applies here: https://github.com/chef/automate/pull/2584/files#r362128435